scripts/document_updater.py

```diff
@@ -0,0 +1,97 @@
+import sys
+import re
+
+from typing import List
+from dataclasses import dataclass
+from pathlib import Path
+from inspect import cleandoc
+
+from exasol_advanced_analytics_framework.example import generator as example_generator
+
+
+class ParseException(Exception):
+    """
+    If input file is not well-formed.
+    """
+
+
+@dataclass
+class Template:
+    path: str
+
+    def render(self):
+        if self.path != "example/generator.py":
+            raise ParseException("document_updater.Template currently only"
```

Comment:
According to its docstring, the ParseException is supposed to be raised if the file is not well-formed. Maybe ValueError will be a better choice here.

Reply:
Fine for me, thanks - see next push.